src/Documenter.jl

```diff
@@ -193,11 +193,19 @@ modules ([`Writers.HTMLWriter`](@ref), [`Writers.LaTeXWriter`](@ref)).
 A guide detailing how to document a package using Documenter's [`makedocs`](@ref) is provided
 in the [Usage](@ref) section of the manual.
 """
-function makedocs(; debug = false, args...)
-    document = Documents.Document(; args...)
+function makedocs(;
+    debug::Bool = false,
+    root::AbstractString = Utilities.currentdir(),
+    source::AbstractString  = "src",
+    pages::Vector = Any[],
+    args...
+)
+    pages = copy_external(pages, root, source)
```

Comment:
I would avoid actually copying files. I would modify the parser part instead. Although that might require refactoring, so I could be OK with copying as well as a temporary solution.

Reply:
I think I'm going to keep it this way for now, it's just a bit more trouble to refactor than it's worth to me. The only downside of this approach is that if for some reason the build fails, there may be some leftover files in `docs/src`.